.env.example

```diff
@@ -41,6 +41,12 @@ EMAIL_SERVER_PORT=
 EMAIL_SERVER_USER=
 EMAIL_SERVER_PASSWORD=
 
+# GoCardless options
```

Comment:
we can't use NEXT_PUBLIC cuz this won't work on docker builds check the notes section in this: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

i recommend to have `GOCARDLESS_ENABLED`  and get it from the server

Reply:
Alright! Good catch, I will change it and have it returned from the server instead.
As you see in one of my dockerfiles, I had to put the public env in the build phase to get it to work. But it did not work in the best way ;)

Reply:
https://github.com/oss-apps/split-pro/pull/132/commits/25fc9faeefd3aadd0b30c1b626ccec66c3506bcb